roles/zenith/defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 # The chart to use
 zenith_chart_repo: https://stackhpc.github.io/zenith
 zenith_chart_name: zenith-server
-zenith_chart_version: 0.1.0-dev.0.main.206
+zenith_chart_version: 0.1.0-dev.0.feature-consul-server-only.209
```

Comment:
can we cut a tag for zenith, then use that tag?

Reply:
Now the Zenith changes have been merged, I have updated this to a `main` version.

Each one of those versions represents a unique commit on `main`, so I'm pretty happy with them TBH. I'd rather not start cutting tags until we have a proper release schedule TBH (not that we don't need to do that, but it needs some thought).